modules/generative-openai/clients/openai.go

```diff
@@ -132,11 +163,30 @@ func (v *openai) Generate(ctx context.Context, cfg moduletools.ClassConfig, prom
 			Result: &trimmedResponse,
 		}, nil
 	}
+
+	message := resBody.Choices[0].Message
```

Comment:
`resBody.Choices` is guaranteed to have always at least 1 entry? is there a chance that this check `len(resBody.Choices) < 1` could be true?

Reply:
Yes, at this point in the code there will always be one entry. otherwise there would be an error.